src/glmfit.jl

```diff
@@ -115,7 +115,23 @@ function confint(obj::AbstractGLM, level::Real)
 end
 confint(obj::AbstractGLM) = confint(obj, 0.95)
 
-deviance(m::AbstractGLM)  = deviance(m.rr)
+deviance(m::AbstractGLM) = deviance(m.rr)
+
+function loglikelihood(m::AbstractGLM)
```

Comment:
@dmbates Funny. :-) The problem is, the idea that "deviance = -2 log L" can be found everywhere on the net, so that's not just an issue with us. Do you have any idea about where I could look to find out the formula? I'd be really interested in understanding and documenting this.